### DISCUSSION TITLE
Incorrect toast notification when marking notification as done

### DISCUSSION THREAD

**Discussion by Lulalaby** [community/community#33432] [Mobile] [answered]
When marking a notification as done, a notification appears telling that "undo done" was successfully 
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.80.0 (78250505)
Device: iPhone 6S (iOS 15.7)
</details>

**Comment by eliperkins** [comment, marked as answer]
Hey @Lulalaby, thanks for this feedback! I agree, the toast text of "Undo Done" is confusing, as it's not clear if an undo of "done" as been completed, or whether tapping on the toast _will undo_ the "done". I'll raise this feedback with the team here and get this addressed such that it's more clear.

**Reply by Lulalaby** [reply]
Thank you
